common/beerocks/bwl/include/bwl/nl80211_client.h

```diff
@@ -72,6 +72,15 @@ class nl80211_client {
      */
     virtual bool get_sta_info(const std::string &local_interface_name,
                               const sMacAddr &sta_mac_address, sStaInfo &sta_info) = 0;
+    /**
+     * @brief Set the tx power limit object
+     *
+     * Set tx power limit for a radio
+     * @param[in] local_interface_name Virtual AP (VAP) interface name.
```

Comment:
I think it works both with radio or virtual interface name

Reply:
I checked on UGW where I have only radio and it works, I didn't check VAP so I'll change the comment to say only the radio name is supported.